analytics/lib/counts.py

```diff
@@ -530,7 +533,8 @@ def do_pull_minutes_active(property: str, start_time: datetime, end_time: dateti
     DependentCountStat('realm_active_humans::day',
                        sql_data_collector(RealmCount, count_realm_active_humans_query, None),
                        CountStat.DAY,
-                       dependencies=['active_users_audit:is_bot:day', '15day_actives::day'])
+                       dependencies=
```

Comment:
We can remove this change; the realm_active_humans::day countstat doesn't have the 1day as a dependency, only the graph does.